### DISCUSSION TITLE
Floorp's Workspace is awesome, but do you know how to use it effectively ?

### DISCUSSION THREAD

**Discussion by gunir** [Floorp-Projects/Floorp#876] [Show and tell]
1st, enable Workspace and Manage Workspace on browser sidebar (recommended, it's the best setup):
![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/dedd8274-09ca-48ed-8281-9e33b67f9629)

Now create new Workspaces using the `+` button on the sidebar, and switch between workspaces using icons on the sidebar comfortably for your work!
![image](https://github.com/Floorp-Projects/Floorp/assets/134402102/95ba4f84-e5cb-4bff-a063-3dec208c5cbd)

Enjoy, Floorp is amazing, this Workspace setup is way way way better than Chrome's Workspace (mostly for looks and feels, but Floorp's is the real deal!).

**Comment by Ryosuke-Asano** [comment]
Great

**Comment by RobLW** [comment]
It's nice. Thank you!
Based on my experience with other browsers, there are some really useful features if it is possible to add to workspaces please:
1. Allow the user to re-order them
2. Possible to increase icon size when managed on the side bar, 
3. allow user to right click on workspace and "sleep all tabs"
4. Maybe use https://fontawesome.com/ for icons?

**Reply by gunir** [reply]
I agree, would be great to be able to reorder, and maybe better, temporarily hide Workspace because sidebar space is limited, so obsolete workspace should be hideable to save space, but if we need to use them again we can just unhide them, because setting up workspace with container kinda take time, so hiding and unhiding is better in this case.

**Reply by anatasiajp** [reply]
@surapunoyousei What do you think about this suggestion ?

I like it.

**Comment by Beryesa** [comment]
Importing from such tab group extensions (i.e. STG) could be really helpful to switch. Also maybe export back if we want to keep a backup or switch to an extension instead of the workspaces feature.

**Comment by Anazark** [comment]
Did this option disappear with an update ?
I'm on 12.1.4 and can't find it anywhere

**Reply by TontonManu** [reply]
Since the last update, workspaces have been accessible from a drop-down menu... when the line you're looking for is at the very bottom, you have to scroll down because the menu is small... this whole thing lacks ergonomics; for me, it's a step backward (requiring multiple clicks instead of just once).
I didn't even find the option to create a keyboard shortcut to switch workspaces, which would have made up for it a bit.

**Reply by tiniestkid** [reply]
@TontonManu Thanks for this information. I discovered Floorp while looking for this exact feature and couldn't find it after installing. Downgraded to 11.30.0 and was able to enable it. Hopefully they re-implement in future updates to Floorp 12.

**Reply by sgtrusty** [reply]
@tiniestkid It's available @ `about:hub#/features/workspaces`, see the docs: https://docs.floorp.app/docs/features/workspaces

**Comment by realaaa** [comment]
thanks for that tip ! very useful indeed, convenient to switch with just one click
